src/libraries/System.Speech/src/Result/SemanticValue.cs

```diff
@@ -9,7 +9,7 @@
 namespace System.Speech.Recognition
 {
     [Serializable]
-    [DebuggerDisplay("'{_keyName}'= {Value}  -  Children = {_dictionary.Count}")]
+    [DebuggerDisplay("'{_keyName}' = {Value}  -  Children = {_dictionary.Count}")]
```

Comment:
```suggestion
    [DebuggerDisplay("{_keyName} = {Value}, Children = {_dictionary.Count}")]
```
`_keyName` is a string so it should already have double quotes.